src/_P078_Eastron.ino

```diff
@@ -166,23 +157,63 @@ boolean Plugin_078(byte function, struct EventStruct *event, String& string)
     case PLUGIN_WEBFORM_LOAD:
       {
         {
-          String options_model[4] = { F("SDM120C"), F("SDM220T"), F("SDM230"), F("SDM630") };
-          addFormSelector(F("Model Type"), P078_MODEL_LABEL, 4, options_model, NULL, P078_MODEL );
+          String options_model[5] = { F("SDM220 & SDM120CT & SDM120"), F("SDM230"), F("SDM72D"), F("DDM18SD"), F("SDM630") };
+          addFormSelector(F("Model Type"), P078_MODEL_LABEL, 5, options_model, NULL, P078_MODEL );
+          addFormNote(F("Submit after changing the modell to update Output Configuration."));
         }
 
         {
           // In a separate scope to free memory of String array as soon as possible
           sensorTypeHelper_webformLoad_header();
-          String options[P078_NR_OUTPUT_OPTIONS];
-          for (int i = 0; i < P078_NR_OUTPUT_OPTIONS; ++i) {
-            options[i] = p078_getQueryString(i);
-          }
-          for (byte i = 0; i < P078_NR_OUTPUT_VALUES; ++i) {
-            const byte pconfigIndex = i + P078_QUERY1_CONFIG_POS;
-            sensorTypeHelper_loadOutputSelector(event, pconfigIndex, i, P078_NR_OUTPUT_OPTIONS, options);
-          }
-        }
+          int model = P078_MODEL;
 
+          if (model == 0) {
+              String options[P078_NR_OUTPUT_OPTIONS_SDM220_SDM120CT_SDM120];
+              for (int i = 0; i < P078_NR_OUTPUT_OPTIONS_SDM220_SDM120CT_SDM120; ++i) {
+                options[i] = p078_getQueryString(i,model);
```

Comment:
These should now be turned into `const __FlashStringHelper* options[]` and the `p078_getQueryString` should return type `const __FlashStringHelper* `

Reply:
... Which is what I now tried to do in the merge of `mega` into this PR.